tests/foreman/ui/test_ansible.py

```diff
@@ -411,3 +411,51 @@ def test_positive_remove_role_in_new_ui(self):
     :expectedresults: The Role is successfully removed from the Host, and no longer shows
         up on the UI
     """
+
+
+# @pytest.mark.stubbed
+@pytest.mark.tier2
+def test_positive_assign_and_remove_role_to_hostgroup_in_new_ui(
```

Comment:
We can ignore mentioning new UI in test name and metadata since it is not new anymore :)
```suggestion
@pytest.mark.tier2
def test_positive_assign_and_remove_role_to_hostgroup(
```

Reply:
Ok, removed the unnecessary word.